main/tests/server/src/com/google/refine/importers/XmlImportUtilitiesTests.java

```diff
@@ -623,41 +551,27 @@ public void processRecordsFromComplexXmlStructure() throws IOException {
     }
 
     // ----------------helpers-------------
-    public void loadSampleXml() {
+    public void loadSampleXml() throws Exception {
         loadData(XmlImporterTests.getSample());
     }
 
-    public void loadSampleJson() {
+    public void loadSampleJson() throws Exception {
         loadData(JsonImporterTests.getSample());
     }
 
-    public void loadData(String xml) {
-        try {
-            inputStream = new ByteArrayInputStream(xml.getBytes("UTF-8"));
-        } catch (UnsupportedEncodingException e1) {
-            Assert.fail();
-        }
+    public void loadData(String xml) throws Exception {
+        inputStream = new ByteArrayInputStream(xml.getBytes("UTF-8"));
     }
 
-    public void ParserSkip() {
-        try {
-            if (parser.current() == TreeReader.Token.Ignorable) {
-                parser.next(); // move parser forward once e.g. skip the START_DOCUMENT parser event
-            }
-        } catch (Exception e1) {
-            Assert.fail();
+    public void ParserSkip() throws Exception {
```

Comment:
I think for these low level helpers where they throw one or two  very specific exceptions, it's probably better to declare the actual exceptions thrown

```suggestion
    public void ParserSkip() throws TreeReaderException {
```